uber/templates/preregistration/preregbase.html

```diff
@@ -125,11 +125,16 @@
                 .slice(index, 1 + index)
                 .addClass('active');
             (type.onClick || $.noop)();
+        };
+        var setKickinFromBadge = function (types, index) {
+            var type = types.options[index];
```

Comment:
Instead of checking that ``type.extra`` is neither ``null`` nor ``undefined``, I would instead check whether it's a number, since that's really what we care about.  You can do that by calling ``_.isNumber(type.extra)``, which is a ``lodash`` function.  I highly recommend lodash in general.

Otherwise +1.